packages/konsole-app/src/registry/RegistryClient.ts

```diff
@@ -38,10 +38,11 @@ export class RegistryClient implements IRegistryClient {
 
   async getRecordTypes (): Promise<IRecordType[]> {
     const records = await this.api?.registry.getResources() ?? [];
-    return records.map(apiRecord => ({
+    const mapped = records.map(apiRecord => ({
       type: apiRecord.messageFqn,
       label: apiRecord.messageFqn
-    })).slice(0, 1); // TODO(marcin): Get unique values instead of taking the first element.
+    }));
+    return Object.values(Object.fromEntries(mapped.map(record => [record.type, record])));
```

Comment:
```suggestion
    
    const distinct = Object.values(Object.fromEntries(mapped.map(record => [record.type, record])));
    
    return distinct;
```

clarifies the intent